catalog/tests/PartitionScheme_unittest.cpp

```diff
@@ -611,6 +665,7 @@ TEST(PartitionSchemeTest, CheckRangePartitionSchemeSerialization) {
     EXPECT_EQ(blocks_in_part_scheme, blocks_in_part_scheme_from_proto);
   }
 }
+*/
```

Comment:
Yes, this test triggers [a debug check](https://travis-ci.org/apache/incubator-quickstep/jobs/213492259#L3473) in protobuf in the debug builds only.